docs/content/latest/explore/ysql-language-features/advanced-features/partitions.md

```diff
@@ -110,6 +142,8 @@ CREATE INDEX ON order_changes_2020_12 (change_date);
 CREATE INDEX ON order_changes_2021_01 (change_date);
 ```
 
+For the implications of creating an index on a partitioned table vs separately on each partition, see [CREATE INDEX](../../../../api/ysql/the-sql-language/statements/ddl_create_index/).
```

Comment:
Suggestion:

... on a partitioned table as opposed to separately ...  (many screen readers cannot process "vs")